### PR TITLE
Holoparasite changes / rebalance

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Instead of just outright reverting holoparasites, this rebalances them slightly, fixes a few issues and makes them not use sound ripped from Jojo.
This more fixes most major issues and changes how some of them works, rather than being an outright balance change (I'd probably nerf healing one a bit more)

### Important/General Changes
 - Copyright sounds ripped straight from Jojo have been removed.
 - Holoparasites will now dust you on death again.
 - Species that have NO_DEATH will now be dusted if they have less than -100 health (Species like zombies cannot die for real. If they are dead and take enough damage now, they will be dusted.)

### Ability Specific Changes
Healing: 
 - Potential matters less for healing (3 to 7 healing instead of 0 to 7.5)

Frenzy:
 - Removed ranged teleport ability
 - Attack will now teleport you behind the victim
 - Cooldown for teleport attack reduced from 3 seconds to 2 + (0.2 * (0 to 5 depending on potential)) (2 to 3 seconds) (potential actually means something now)
 - The Attack() function of frenzy no longer calls itself and therefor no longer bursts the eardrums of the guardian.
 - Reduces the range of the throw to 2 tiles rather than 20, which only takes effect if the potential is greater than 3 (B or A).

Time Paradox:
 - Renamed to Time Distortion
 - Cost lowered from 6 to 4
 - Decoys spawn on guardian attack rather than as an ability
 - Decoy spawn time changed to 6 seconds
 - Recharge time changed to 8 to 13 seconds
 - Removed godmode effect, this now just spawns decoys rather than giving immortality for a while
 - Doppelgangers now fade out slowly (which is just a cool effect)

The Hand:
 - No longer stuns moved targets

## Why It's Good For The Game

Rebalances and fixes many issues with holoparasites, without reverting them completely.
Fixes a load of holoparasite issues.
Removes copyrighted sound effects.

## Changelog
:cl:
sounddel: Removes 5 sounds that were ripped from Jojo
balance: Healing holoparasite ability potential matters less for healing (3 to 7 healing instead of 0 to 7.5)
balance: Frenzy Holopara - Removed ranged teleport ability
balance: Frenzy Holopara - Attack will now teleport you behind the victim
balance: Frenzy Holopara - Cooldown for teleport attack reduced from 3 seconds to 2 + (0.2 * (0 to 5 depending on potential)) (2 to 3 seconds) (potential actually means something now)
balance: Frenzy Holopara - The Attack() function of frenzy no longer calls itself and therefor no longer bursts the eardrums of the guardian.
balance: Frenzy Holopara - Reduces the range of the throw to 2 tiles rather than 20, which only takes effect if the potential is greater than 3 (B or A).
balance: Time Holopara - Renamed to Time Distortion
balance: Time Holopara - Cost lowered from 6 to 4
balance: Time Holopara - Decoys spawn on guardian attack rather than as an ability
balance: Time Holopara - Decoy spawn time changed to 6 seconds
balance: Time Holopara - Recharge time changed to 8 to 13 seconds
balance: Time Holopara - Removed godmode effect, this now just spawns decoys rather than giving immortality for a while
balance: Time Holopara - Doppelgangers now fade out slowly (which is just a cool effect)
balance: Hand Holopara - No longer stuns mobs
/:cl:

### CODE REVIEW

File: code/modules/guardian/abilities/major/time.dm [hunk 1]
```diff
@@ -1,98 +1,81 @@
 /datum/guardian_ability/major/time
-	name = "Time Erasure"
-	desc = "The guardian can erase a short period of time."
+	name = "Time Distoration"
+	desc = "Distorts time and spawn causing fragments from other timelines to appear as distractions."
```
Comment (PowerfulBacon) [inline code/modules/guardian/abilities/major/time.dm]:
```suggestion
	desc = "Distorts time and space, causing fragments from other timelines to appear as distractions."
```

**Comment by zeskorion** [comment]
i think explosive needs some looking into as well

**Comment by TeomanTheGreat** [comment]
Bacon out of curiosity have you even watched jojo?

**Comment by PowerfulBacon** [comment]
> Bacon out of curiosity have you even watched jojo?

No, it's some shit anime and gameplay > some shitty reference

**Review by ike709 — APPROVED** [review]

**Comment by ike709** [comment]
YOLO

**Comment by Absolucy** [comment]
i'm late (just digging thru old PRs to see what's different with holoparas compared to other servers), but smh, the frenzy sounds aren't from jojo

**Pull request merged**